Tests/Features/Page/seo.feature

```diff
@@ -33,6 +33,24 @@ Feature: Create a seo
         Given I am on "/en/jedi-profile-anakin"
         Then the title should be "Master of the Force - Anakin"
 
+        Given I am on "/fr/victoire-dcms/business-template/show/4"
+        And I select the option "SEO" in the dropdown "Page"
+        Then I should see "Avancé"
+        Then I follow "Avancé"
+        And I check "URL indexée"
```

Comment:
I think you forgot to submit the form.
Also, I suggest to validate the sitemap.xml before and after the submit to be sure there is a link between the actions :)

Can you please add too at least one test to validate that BusinessPages (and VirtualBusinessPages) are also well managed ?